Sming/Libraries/modbusino/samples/generic/app/application.cpp

```diff
@@ -0,0 +1,24 @@
+#include <SmingCore.h>
+#include <Debug.h>
+#include <Modbusino.h>
+
+#define ARRLEN 3
+uint16_t mbDataArray[ARRLEN] = {0,0,0};
+ModbusinoSlave MbSlave(MB_SLAVE_ADDR, mbDataArray, ARRLEN);
```

Comment:
Rename `MbSlave` to `mbSlave` to match the [coding style](https://sming.readthedocs.io/en/latest/contribute/coding-style.html).